### PR TITLE
Align buttons on Edit Event

### LINKED ISSUE #1299: The buttons in the 'Edit Event' modal are improperly aligned.
**Describe the bug**

When accessing the modal to edit an event, the buttons within the interface appear to be improperly aligned, resulting in a visual inconsistency and potentially impacting the user experience.

**To Reproduce**

1. Log in to the Talawa admin panel.
2. Open an organization from the dashboard.
3. Navigate to the event screen.
4. Click on an existing event in the event calendar. If there are no events, create a new one and click on it.
5. Inside the event details, click on the edit icon within the edit modal.

**Expected behavior**

Proper alignment of buttons.

**Actual behavior**

Buttons are improperly aligned.

**Screenshots**
image

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by PriyadharshiniNagarathinam**
**What kind of change does this PR introduce?**

Align buttons in Edit Event component

**Issue Number:**

Fixes #1299 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

Talawa Events - Google Chrome 06-01-2024 15_41_32

**If relevant, did you update the documentation?**

No

**Summary**

Align the buttons of Edit Event Component

**Does this PR introduce a breaking change?**

No

**Other information**

NA

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -95,6 +95,9 @@
         "jest-preview": "^0.3.1",
         "jquery": "^3.2.1",
         "postcss-modules": "^6.0.0"
+      },
+      "engines": {
```
Comment (rishav-jha-mech) [inline package-lock.json:99]: Revert any change in this file

File: src/components/EventListCard/EventListCard.module.css [hunk 2]
```diff
@@ -88,13 +88,19 @@
...
 }
 .checkboxdiv > label > input {
   margin-left: 10px;
 }
-
+.dispflex {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 5px;
+  margin-right: 5px;
```
Comment (rishav-jha-mech) [inline src/components/EventListCard/EventListCard.module.css:101]: Refactor this into a single line
```css
margin: 10px 5px 5px unset;
```

File: src/components/EventListCard/EventListCard.module.css [hunk 3]
```diff
@@ -103,9 +109,14 @@
 }
 .checkboxdiv {
   display: flex;
+  flex-direction: column;
 }
-.checkboxdiv > div {
+/* .checkboxdiv > div {
```
Comment (rishav-jha-mech) [inline src/components/EventListCard/EventListCard.module.css:114]: Don't include any commented code for the PR

File: src/screens/LoginPage/LoginPage.tsx [hunk 4]
```diff
@@ -233,8 +233,14 @@ function loginPage(): JSX.Element {
         <Row className={styles.row}>
           <Col sm={0} md={6} lg={7} className={styles.left_portion}>
             <div className={styles.inner}>
-              <PalisadoesLogo className={styles.palisadoes_logo} />
-              <p className="text-center">{t('fromPalisadoes')}</p>
+              <a
+                href="https://www.palisadoes.org/"
+                target="_blank"
```
Comment (rishav-jha-mech) [inline src/screens/LoginPage/LoginPage.tsx:238]: This file is unrelated to your PR, revert any change you made to this file.

File: src/screens/UserPortal/UserLoginPage/UserLoginPage.tsx [hunk 5]
```diff
@@ -29,14 +29,21 @@ export default function userLoginPage(): JSX.Element {
...
-        <img
-          className={styles.palisadoesImage}
-          src={PalisadoesImage}
-          alt="Palisadoes Branding"
-        />
-        <h6 style={{ textAlign: `center` }}>
-          <p>{t('fromPalisadoes')}</p>
-        </h6>
+        <a
+          href="https://www.palisadoes.org/"
```
Comment (rishav-jha-mech) [inline src/screens/UserPortal/UserLoginPage/UserLoginPage.tsx:33]: This file is unrelated to your PR, revert any change you made to this file.

**Comment by palisadoes** [comment]
Please fix the conflicting files

File: src/screens/LoginPage/LoginPage.tsx (hunk 4 above)
Comment (PriyadharshiniNagarathinam) [reply src/screens/LoginPage/LoginPage.tsx]: I reverted and modified as you said can you please review this again??

**Comment by PriyadharshiniNagarathinam** [comment]
@palisadoes @rishav-jha-mech I fixed the issues can you please review again?

**Comment by rishav-jha-mech** [comment]
@PriyadharshiniNagarathinam code looks okay, please share screenshots from Mobile View, Tab View and Laptop View so that we can verify the UI and get your PR merged

**Comment by aashimawadhwa** [comment]
@PriyadharshiniNagarathinam  any updates?

**Comment by aashimawadhwa** [comment]
@palisadoes  can we assign this issue to someone else?

**Comment by palisadoes** [comment]
Closing due to inactivity
